src/position-manager.ts

```diff
@@ -0,0 +1,146 @@
+import { ethereum, BigInt, Address } from "@graphprotocol/graph-ts";
+import {
+  Collect,
+  DecreaseLiquidity,
+  IncreaseLiquidity,
+  NonfungiblePositionManager,
+  Transfer,
+} from "../generated/NonfungiblePositionManager/NonfungiblePositionManager";
+import { ERC20Token, PaprController, Position } from "../generated/schema";
+import { handleSyntheticSwapFromLP } from "./activity";
+import { loadOrCreateERC20Token } from "./utils";
+
+const paprControllerAddress = "0x3b29c19ff2fcea0ff98d0ef5b184354d74ea74b0";
+
+function isPaprPool(
```

Comment:
theoretically we'd want to track all papr pools, but we can save that complexity for another day